packages/admin/docs/03-pages/01-getting-started.md

```diff
@@ -48,6 +48,12 @@ protected static ?string $navigationLabel = 'Custom Navigation Label';
 protected static ?string $slug = 'custom-url-slug';
 ```
 
+You may override the [maximum content width](https://filamentphp.com/docs/2.x/admin/appearance#changing-the-maximum-content-width) of a page using the `$maxContentWidth` property:
```

Comment:
These docs probably belong with the other max width docs instead of here

Reply:
What you think is better. My reasoning was that this is part of "customizing a page", so that's why I put it here.